src/workflows/airqo_etl_utils/satellilte_utils.py

```
@@ -0,0 +1,160 @@
+            val_df = device_df[device_df["date"].dt.month.isin(val_months)]
+            test_df = device_df[device_df["date"].dt.month.isin(test_months)]
+
+            train_data = pd.concat([train_data, train_df])
+            validation_data = pd.concat([validation_data, val_df])
+            test_data = pd.concat([test_data, test_df])
+
+        train_data.drop(columns=["date", "ID"], axis=1, inplace=True)
+        validation_data.drop(columns=["date", "ID"], axis=1, inplace=True)
+        test_data.drop(columns=["date", "ID"], axis=1, inplace=True)
+
+        train_target, validation_target, test_target = (
```

Ruff

Local variable `train_target` is assigned to but never used

Remove assignment to unused variable `train_target`

(F841)